wiki/608223-2014i2b2NLPDeidChallengeDataset.md

```diff
@@ -0,0 +1,63 @@
+## **The i2b2 challenge dataset**
+
+2014 i2b2 NLP De-id Challenge aimed at developing automated methods to detect the location and types of sensitive information in clinical notes. [[1]](#1).
+
+### **Annotation types**
+2014 i2b2 challenge datasets included clinical notes for 296 diabetic patients drawn from the Research Patient Data Repository of Partners Healthcare. [[2]](#2) [[3]](#3) The challenge organizers annotated the type and location of the sensitive information. The annotations were mostly compliant to Safe Harbor HIPAA standard with 18 categories but also were expanded to include some non-HIPAA identifiers that could potentially leave a footprint for re-identification of patients. e.g. the city where the patient locates.
+
+*Table 1. shows a list of annotation types defined by the 2014 i2b2 challenge with specification on whether the annotations belong to HIPAA PhIs and the existance of the annotations in 2014 i2b2 datasets.*
```

Comment:
Similarly to a manuscript, make sure that there is at least one reference to a Table or Figure. In other words, don't add a Table or Figure that is not used/discussed in the text.